config/initializers/assets.rb

```diff
@@ -2,6 +2,8 @@
 
 # Version of your assets, change this if you want to expire all your assets.
 Rails.application.config.assets.version = '1.0'
+Date::DATE_FORMATS[:default] = "%d-%m-%Y"
```

Comment:
To add Date formats, it's better to use I18n Rails mechanisms.
See this section of the I18n guide for details: http://guides.rubyonrails.org/i18n.html#adding-date-time-formats :)